src/main/resources/templates/db_scripts/all_finance_data_view.sql

```diff
@@ -1,5 +1,6 @@
 CREATE OR REPLACE VIEW ${myuniversity}_${mymodule}.finance_data_view AS
 SELECT
+  DISTINCT ON (fiscal_year.id, fund.id, budget.id)
```

Comment:
The same condition as in BE side: https://github.com/folio-org/mod-finance/blob/master/src/main/java/org/folio/services/financedata/FinanceDataValidator.java#L63
All these fields are primary keys where Postgres creates automatically unique indexes, so should not influence performance very much.
ORDER BY is added to keep best practices to define ORDER BY with DISTINCT